docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>You can get rid of the Theano warning with this at the top of the NB, with other imports:</p><pre class="ql-syntax" spellcheck="false">import warnings
warnings.simplefilter(action="ignore", category=FutureWarning)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
<p>😂</p>